libs/@local/hash-graphql-shared/src/queries/entity.queries.ts

```diff
@@ -0,0 +1,68 @@
+import { gql } from "@apollo/client";
```

Comment:
Can we delete the existing one of these in the frontend and import this one instead, to avoid duplicating it?

Reply:
Ah yes, forgot to do this. Have done so in https://github.com/hashintel/hash/pull/3367/commits/f237d7120c807777e5feb4eb05b2cd17022f34fd